src/lib/packager/clients/repositories.rb

```diff
@@ -2046,5 +2057,17 @@ def plugin_service_check(service_alias, msg)
       Popup.Message(msg)
       false
     end
+
+    # Shows a warning message when repository managed by a service
+    # @param [Hash] sourceState the current state of the repository or service
+    def warn_service_repository(source_state)
+      msg = _("Repo %{name} is managed by service %{service}.\n"\
```

Comment:
NP: Maybe full `Repository` should be used here instead of short (and informal) `Repo`.

Reply:
What about `'%{name}' repository is manager by service '%{service}'`